timescaledb/how-to-guides/hypertables/about-hypertables.md

```diff
@@ -0,0 +1,140 @@
+# About hypertables
+Hypertables are PostgreSQL tables with special features that make it easy to
+work with time-series data. You interact with them just as you would with
+regular PostgreSQL tables. But behind the scenes, hypertables automatically
+partition your data into chunks by time.
+
+In TimescaleDB, hypertables exist alongside regular PostgreSQL tables. Use
+hypertables to store time-series data. This gives you improved insert and query
+performance, and access to useful time-series features. Use regular PostgreSQL
+tables for other relational data.
+
+## Hypertable partitioning
+When you create and use a hypertable, it automatically partitions data by time,
+and optionally by space.
+
+Each hypertable is made up of child tables called chunks. Each chunk is assigned
+a range of time, and only contains data from that range. If the hypertable is
+also partitioned by space, each chunk is also assigned a subset of the space
+values.
+
+### Time partitioning
+Each chunk of a hypertable only holds data from a specific time range. When you
+insert data from a time range that doesn't yet have a chunk, TimescaleDB
+automatically creates a chunk to store it.
+
+By default, each chunk covers 7 days. You can change this to better suit your
+needs. For example, if you set `chunk_time_interval` to 1 day, each chunk stores
+data from the same day. Data from different days is stored in different chunks. 
+
+<img class="main-content__illustration"
+src="https://s3.amazonaws.com/assets.timescale.com/docs/images/getting-started/hypertables-chunks.png"
+alt="A normal table compared to a hypertable. The normal table holds data for 3 different days in one container. The hypertable contains there containers, called chunks, each of which holds data for a separate day." />
+
+<highlight type="note">
+TimescaleDB divides time into potential chunk ranges, based on the
+`chunk_time_interval`. If data exists for a potential chunk range, that chunk is
+created.
+
+In practice, this means that the start time of your earliest chunk doesn't
+necessarily equal the earliest timestamp in your hypertable. Instead, there
+might be a time gap between the start time and the earliest timestamp. This
+doesn't affect your usual interactions with your hypertable, but might affect
+the number of chunks you see when inspecting it.
+</highlight>
+
+#### Best practices for time partitioning
```

Comment:
I feel like four heading levels is a little too much. Can we bring this back a bit somehow?